__tests__/index.test.js

```diff
@@ -94,19 +94,33 @@ describe("Purchases", () => {
     expect(listener).toHaveBeenCalledTimes(0);
   });
 
-  it("calling configure with something other than string throws exception", () => {
+  it("calling configure without an object throws exception", () => {
     const Purchases = require("../dist/index").default;
 
     expect(() => {
-      Purchases.configure("api_key", 123)
+      Purchases.configure("api_key")
+    }).toThrowError("Invalid API key. It must be called with an Object: confifure({apiKey: \"key\"})");
```

Comment:
This will prevent users from simply renaming `setup` to `configure` when coming from version 4, without using the new `Object` syntax /cc @vegaro